.jenkins/pipelines/OpenEnclave/releases/publish/Jenkinsfile

```diff
@@ -16,59 +16,42 @@ library "OpenEnclaveJenkinsLibrary@${params.OECI_LIB_VERSION}"
 /**
  * Publish OpenEnclave Ubuntu packages to the Microsoft APT repository
  *
- * @param release_version  The version of the Open Enclave release to install
+ * @param release_version  The version of the Open Enclave release to publish
  * @param os_release       The distribution version without "." (e.g. 1804)
  */
 
 def ubuntuPublishRelease(String release_version, String os_release) {
 
     helpers.WaitForAptLock()
     withCredentials([file(credentialsId: 'azure-repoapi-client', variable: 'repoapiclient'),
-                     string(credentialsId: 'msft-apt-repo-pem', variable: 'repoclientpem'),
-                     string(credentialsId: 'ReleaseAADClientId', variable: 'AADClientId'),
-                     string(credentialsId: 'ReleaseAADResource', variable: 'AADResource'),
-                     string(credentialsId: 'ReleaseAADTenant', variable: 'AADTenant'),
-                     string(credentialsId: "MicrosoftAptRepositoryUbuntu${os_release}", variable: "Ubuntu${os_release}repoid")]) {
+                     file(credentialsId: "ubuntu-${os_release}-repo-config", variable: 'aptconfig'),
```

Comment:
For the time being, this file is uploaded directly to Jenkins

Reply:
Could you explain the benefit of doing this via generating it in the job?